pkgs/by-name/ri/rime-ls/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  librime,
+  rime-data,
+  symlinkJoin,
+  rimeDataPkgs ? [ rime-data ],
```

Comment:
Since this is only used at test time, maybe it doesn't have to be an option?

Reply:
Ah, I forgot to remove this option, thanks for reminding me.